### PR TITLE
clio-30516 thSelect Visual fixes

### PR DESCRIPTION
**Pull request by JefCurtis**
- Add a 'condensed' attr to th-input and th-select with tests. 
- Update sass files for both replacing rem units with px. 
- Changed with-label's margin-bottom from 6px to 8px. 
- Add new examples for the condensed attr.

### CODE REVIEW

File: themis_components/thInput/index.scss [hunk 1]
```diff
@@ -1,17 +1,13 @@
 .th-input {
   outline: none;
-  -webkit-box-flex: 1;      // OLD - iOS 6-, Safari 3.1-6
-  -moz-box-flex: 1;         // OLD - Firefox 19-
-  -webkit-flex: 1;          // Chrome
-  -ms-flex: 1;              // IE 10
-  flex: 1;                  // NEW, Spec - Opera 12.1, Firefox 20+
```
Comment (JefCurtis) [inline themis_components/thInput/index.scss]: We are using `gulp-autoprefixer` to take care of this for us.

Comment (seanhealy) [inline themis_components/thInput/index.scss]: :white_check_mark: Resolved

File: themis_components/thInput/index.scss [hunk 2]
```diff
@@ -1,17 +1,13 @@
...
+  flex: 1;
   width: 100%;
-  font-size: 1.25rem;
+  font-size: 14px;
```
Comment (seanhealy) [inline themis_components/thInput/index.scss]: :white_check_mark: Hurray!!!

File: themis_components/thInput/thInput.directive.spec.coffee [hunk 3]
```diff
@@ -9,7 +9,8 @@ describe 'thInput', ->
         <th-input type="text"
                   name="nickname"
                   value="Steve-o"
-                  placeholder="Enter your nickname">
+                  placeholder="Enter your nickname"
+                  condensed="true">
```
Comment (seanhealy) [inline themis_components/thInput/thInput.directive.spec.coffee]: The `>` here should be on the following line.

File: themis_components/thInput/thInput.template.html [hunk 4]
```diff
@@ -1,16 +1,21 @@
...
   <span ng-if="input.prefix" class="th-input-prefix">{{input.prefix}}</span>
   <input
     ng-model="input.model"
     class="th-input"
-    ng-class="[{'with-icon': input.icon}, {'with-prefix': input.prefix}, {'with-postfix': input.postfix}]"
+    ng-class="[
+      {'with-icon': input.icon},
+      {'with-prefix': input.prefix},
+      {'with-postfix': input.postfix}
+    ]"
```
Comment (seanhealy) [inline themis_components/thInput/thInput.template.html]: :white_check_mark: Yay for shorter lines. :D

File: themis_components/thWithLabel/index.scss [hunk 5]
```diff
@@ -21,7 +21,7 @@ label.th-label {
     color: $light-grey;
     font-weight: bold;
     margin-left: 4px;
-    margin-bottom: 6px;
+    margin-bottom: 8px;
```
Comment (seanhealy) [inline themis_components/thWithLabel/index.scss]: Should this be effected by `condensed`?

**Comment by seanhealy** [comment]
Do you have a second ticket for the work on `thInput`?

File: themis_components/thInput/readme.md [hunk 6]
```diff
@@ -11,6 +11,7 @@ Add your input fields with a little style.
   - icons
   - postfix
   - prefix
+  - condensed
```
Comment (seanhealy) [inline themis_components/thInput/readme.md]: I like how the other readme specifies the defaults. Want to do that here too?

File: themis_components/thWithLabel/index.scss (hunk 5 above)
Comment (JefCurtis) [inline themis_components/thWithLabel/index.scss]: Hmm good point. It's not in Themis now but should it?

File: themis_components/thInput/readme.md (hunk 6 above)
Comment (JefCurtis) [inline themis_components/thInput/readme.md]: :+1:

File: themis_components/thWithLabel/index.scss (hunk 5 above)
Comment (seanhealy) [inline themis_components/thWithLabel/index.scss]: Perhaps check with design. I would say likely yes. :)

File: themis_components/thInput/thInput.directive.spec.coffee (hunk 3 above)
Comment (seanhealy) [inline themis_components/thInput/thInput.directive.spec.coffee]: :white_check_mark: Resolved

File: themis_components/thInput/readme.md (hunk 6 above)
Comment (seanhealy) [inline themis_components/thInput/readme.md]: :white_check_mark: Resolved

File: themis_components/thWithLabel/index.scss (hunk 5 above)
Comment (eugeneor) [inline themis_components/thWithLabel/index.scss]: What would the ratio be when `condensed` is applied to the padding?

Comment (JefCurtis) [inline themis_components/thWithLabel/index.scss]: @eugeneor It's roughly 20% thinner with the `condensed` flag.

Comment (JefCurtis) [inline themis_components/thWithLabel/index.scss]: So after chatting with @eugeneor we are gonna stick with the same margin between label and input field regardless of the `condensed` flag. :+1:

Comment (seanhealy) [inline themis_components/thWithLabel/index.scss]: :white_check_mark: Resolved

**Pull request merged**